vscode/src/custom-prompts/utils/menu.ts

```diff
@@ -212,3 +198,30 @@ export const CustomCommandConfigMenuItems = [
     { kind: -1, id: 'separator', label: '' },
     { kind: 0, label: 'See Example Commands', id: 'example', type: 'default' },
 ]
+
+export async function showAskQuestionQuickPick(): Promise<string> {
+    const quickPick = vscode.window.createQuickPick()
+    quickPick.title = '/ask'
+    quickPick.placeholder = 'Your question'
```

Comment:
```suggestion
    quickPick.placeholder = 'Enter your question for Cody here...'
```

To align with the placeholder text in the other menus?
<img width="664" alt="image" src="https://github.com/sourcegraph/cody/assets/68532117/a7530cef-aa48-45f8-a375-47f5100ee3e9">

Reply:
Updated https://github.com/sourcegraph/cody/pull/798/commits/c0f1e765bafe39bc63e969c6d977214c76c2d9f8 according to the [design](https://www.figma.com/file/9112BsKsJc1BpO2j8XYwLL/%F0%9F%A4%96-Cody-VS-Code-%5BMain%5D?node-id=4560:13972&mode=dev).